test/image/mocks/image_adventurer.json

```diff
@@ -1 +1 @@
9, 255], [194, 200, 240, 255], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [143, 94, 71, 254], [95, 67, 45, 254], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [97, 65, 43, 254], [128, 77, 54, 254], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0]], [[0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [194, 200, 240, 255], [133, 144, 159, 255], [165, 174, 234, 255], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [143, 94, 71, 254], [128, 77, 54, 254], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [97, 65, 43, 254], [128, 77, 54, 254], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0]], [[0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [194, 200, 240, 255], [194, 200, 240, 255], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [127, 78, 55, 254], [128, 77, 54, 254], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [97, 65, 43, 254], [128, 77, 54, 254], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0]], [[0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0]]], "x0": 0.5, "y0": 0.5, "colormodel": "rgba", "type": "image"}], "layout": {"yaxis": {"dtick": 1, "range": [40, 7], "zeroline": false}, "xaxis": {"dtick": 1, "zeroline": false}, "width": 400, "height": 400, "margin": {"t": 0, "b": 0, "l": 0, "r": 0}}}
```

Comment:
Curious to know how these numbers for the mock are achieved?
Did you read the raw data from a raw bitmap file (i.e. `.bmp`) using cpp? 
Or decoded a `.png` with javascript or python? Or...

Reply:
@antoinerg what did you change in this JSON file?

Reply:
@etpinard I removed the `scaleanchor` that was initially present since it is now the default. This way, this mock also tests the smart default!

Reply:
> Curious to know how these numbers for the mock are achieved?
> Did you read the raw data from a raw bitmap file (i.e. `.bmp`) using cpp?
> Or decoded a `.png` with javascript or python? Or...

@archmoj I read the a `png` using the `imread`  function in Python:
```python
#! /usr/bin/env nix-shell
#! nix-shell -i python3 -p python3 python3Packages.scikitimage -p python3 python3Packages.numpy
import json
from skimage.io import imread
import numpy as np
# rgb_img = imread('small.jpg')

rgb_img = imread('/home/antoine/Downloads/Individual
Sprites/adventurer-idle-2-00.png')
# rgb_img[:, :, 3] = rgb_img[:, :, 3] / 255
figure = {
    "data": [{
        "z": rgb_img.tolist(),
        "x": 0.5,
        "y": 0.5,
        "colormodel": "rgba",
        "type": 'image'
    }],
    "layout": {
        "yaxis": {
            "scaleanchor": "x",
            "dtick": 1,
            "range": [-40, -5],
            "zeroline": False
        },
        "xaxis": {
            "dtick": 1,
            "zeroline": False
        },
        "width": 400,
        "height": 400,
        "margin": {"t": 0, "b": 0, "l": 0, "r": 0}
    }
}
json = json.dumps(figure)
print(json)
```